src/fidesops/ops/schemas/connection_configuration/connection_secrets_email.py

```diff
@@ -0,0 +1,19 @@
+from typing import List, Optional
+
+from fidesops.ops.schemas.base_class import NoValidationSchema
+from fidesops.ops.schemas.connection_configuration.connection_secrets import (
+    ConnectionConfigSecretsSchema,
+)
+
+
+class EmailSchema(ConnectionConfigSecretsSchema):
+    """Schema to validate the secrets needed for the EmailConnector"""
+
+    to_email: str
+    test_email: Optional[str]  # Email to send a connection test email
+
+    _required_components: List[str] = ["to_email"]
```

Comment:
currently I'm thinking the user can just configure the single email that should be sent out (or we could do a list of emails, that's in an earlier commit).  Additionally, if they have a "test_email" configured, I think they could send an email to themselves if they wanted with the same template they'd send to the user, just without the data in it, so they could preview if they wished.

Reply:
Current design just has this separately creating the single `EmailConfig` that Catherine developed, and using that resource to do the sending part.

Reply:
To one `email` address is a fair assumption for now! And good thoughts on the test email for testing the connector.